website/notifications/views.py

```diff
@@ -73,6 +73,12 @@ def configure_subscription(auth):
             sentry.log_message('{!r} attempted to subscribe to private node, {}'.format(user, target_id))
             raise HTTPError(http.FORBIDDEN)
 
+        if isinstance(node, Registration):
+            sentry.log_message(
+                '{!r} attempted to subscribe to registration, {}'.format(user, target_id)
+            )
+            raise HTTPError(http.BAD_REQUEST)
```

Comment:
A POST to api/v1/subscriptions/ with a registration in the request body could cause a NotificationSubscription to be attached to a registration (saw no places where this was possible in the front end, but I guess it could be called directly).

Reply:
Do you know if `log_message()` will include the call stack? If this does ever hit, it would be good to have the context. I did a quick check of our code vs the sentry docs, and I'm not sure. (Research research) Yeah, looks like it does. [This](https://sentry2.cos.io/cos/osf-back-end/issues/322/) is one of ours, and it's well documented. Very good. Including just in case you also hadn't known if this were the case.

Reply:
ah, thanks @brianjgeiger !